src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+const largeImage = document.getElementById('largeImg');
+
+// eslint-disable-next-line no-shadow
+gallery.addEventListener('click', (event) => {
+  event.preventDefault();
```

Comment:
The use of `event.preventDefault()` here prevents the default action of the link, which is likely intended to stop the page from navigating when a thumbnail is clicked. Ensure that this behavior is desired, as it will prevent any link navigation within the gallery.